components/Input.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChangeEvent, FocusEvent, useEffect, useState } from "react";
+import { IInputInformationModel } from "../models/inputInformationModel";
+import { IUserInformation } from "../models/userInformation";
+import { IValidation } from "../models/validationModel";
+import styles from "../styles/UserForms.module.css";
+
+interface IInputProps {
+  inputInformation: IInputInformationModel;
+  //handleOnBlur(e: FocusEvent<HTMLInputElement, Element>): void;
```

Comment:
Removed uncommented code. It could lead to bugs in the future due to it getting stale quickly (e.g. someone uncommenting it and expecting it to work with all the other changes made in the code)

Reply:
done